cglib/src/main/java/net/sf/cglib/proxy/BridgeMethodResolver.java

```diff
@@ -32,9 +32,9 @@
 import org.objectweb.asm.Opcodes;
 
 /**
- * Uses bytecode reflection to figure out the targets of all bridge methods
- * that use invokespecial, so that we can later rewrite them to use invokevirtual.
- * 
+ * Uses bytecode reflection to figure out the targets of all bridge methods that use invokespecial
+ * and invokeinterface, so that we can later rewrite them to use invokevirtual.
```

Comment:
can you clarify this to explain why we need to find invokeinterface (basically the description from the PR)?